src/components/Loading/Loading.module.scss

```
@@ -0,0 +1,17 @@
+.loading-img {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 0.9);
+  z-index: 99;
+
+  img {
+    width: 200px;
+    height: 200px;
+  }
+}
```

top값과 left값을 사용하실거면 이렇게 쓰는게 좀 더 보기 좋을거 같습니다!
```suggestion
.loading-img {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: rgba(255, 255, 255, 0.9);
  z-index: 99;

  img {
    width: 200px;
    height: 200px;
  }
}
```

---

감사합니다! 저런방법으로도 작성해 봐야겠어요